### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by cs-adarsh-gupta**
### Descriptions:
Fixed and Edited the connector as per FSR standards

#### Affected Actions:
- [ ] All

#### UTCs:
- [ ] Connector installation verified
- [ ] Connector logo verified
- [ ] Check health verified
- [ ] Docs link verified
- [ ] Actions and Playbooks list verified
- [ ] Playbook tags verified
- [ ] All playbooks are in info mode verified
- [ ] All playbooks are in inactive mode verified
- [ ] Ingestion playbooks are verified
- [ ] Verified platform dependencies remain unaffected

### CODE REVIEW

File: proofpoint-trap/info.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+          "name": "alertSourceID",
+          "tooltip": "Specify the Alert Source ID of which you want to create the alert",
+          "description": "Specify the Alert Source ID of which you want to create the alert",
+          "required": true,
+          "visible": true,
+          "editable": true
+        },
+        {
+          "title": "Data",
+          "type": "object",
```
Comment (cs-utkarsh-surana) [inline proofpoint-trap/info.json:68]: Check the type of parameter

File: proofpoint-trap/info.json [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        "quarantine_results": [],
+        "successful_quarantines": "",
+        "failed_quarantines": "",
+        "pending_quarantines": ""
+      }
+    },
+    {
+      "operation": "get_incidents",
+      "title": "Get Incidents",
+      "description": "Retrieves All Incident details based on the parameters provided",
```
Comment (cs-utkarsh-surana) [inline proofpoint-trap/info.json]: All --> all

Comment (cs-utkarsh-surana) [inline proofpoint-trap/info.json]: make the description correction in doc file as well

File: proofpoint-trap/info.json [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+          "type": "select",
+          "name": "state",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "value": "closed",
+          "tooltip": "Select state of the incident",
+          "description": "Select state of the incident",
+          "options": [
+            "new",
```
Comment (cs-utkarsh-surana) [inline proofpoint-trap/info.json]: Change the options to "New", Open etc for better UI experience.

File: proofpoint-trap/info.json [hunk 4]
```diff
@@ -0,0 +1,194 @@
+{
+  "name": "proofpoint-trap",
```
Comment (cs-utkarsh-surana) [inline proofpoint-trap/info.json:2]: Check images

File: proofpoint-trap/info.json [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+  "version": "1.0.0",
```
Comment (cs-utkarsh-surana) [inline proofpoint-trap/info.json:3]: requirement.txt file can be removed or else add the copy right details.

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

File: proofpoint-trap/info.json (hunk 1 above)
Comment (cs-adarsh-gupta) [reply proofpoint-trap/info.json:68]: Its correct it behaves as JSON, I verified it after installing it in FSR

File: proofpoint-trap/info.json (hunk 2 above)
Comment (cs-adarsh-gupta) [reply proofpoint-trap/info.json]: Sure

Comment (cs-adarsh-gupta) [reply proofpoint-trap/info.json]: Sure

File: proofpoint-trap/info.json (hunk 3 above)
Comment (cs-adarsh-gupta) [reply proofpoint-trap/info.json]: Sure

File: proofpoint-trap/info.json (hunk 5 above)
Comment (cs-adarsh-gupta) [reply proofpoint-trap/info.json:3]: Removed

File: proofpoint-trap/info.json (hunk 4 above)
Comment (cs-adarsh-gupta) [reply proofpoint-trap/info.json:2]: Changing

**Pull request merged**